edge-functions/svelte-middleware/README.md

```diff
@@ -0,0 +1,21 @@
+# SvelteKit With Middleware
+
+This example shows how use a Vercel Middleware in Sveltekit
+
+## Demo
+
+https://svelte-middleware-guivercel.vercel.sh/
+
+## How to Use
+
+- Install the Vercel CLI `npm i -g vercel`
+- Install degit `npm i -g degit`
+- Clone the repository `degit https://github.com/vercel/examples/edge-functions/svelte-middleware svelte-middleware`
```

Comment:
this doesn't seem to actually work. degit only understands the repo URL itself, not subdirectories (it should probably error on encountering an argument like this)

```suggestion
- Install the Vercel CLI `npm i -g vercel`
- Clone the repository `npx degit https://github.com/vercel/examples vercel-examples`
- Navigate to the directory `cd vercel-examples/edge-functions/svelte-middleware`
```